build_tools/python/e2e_model_tests/run_module_utils_test.py

```diff
@@ -0,0 +1,75 @@
+## Copyright 2022 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+import unittest
+
+from e2e_model_tests import run_module_utils
+from e2e_test_framework.definitions import common_definitions, iree_definitions
+from e2e_test_framework.device_specs import device_parameters
+
+
+class RunModuleTuilsTest(unittest.TestCase):
+
+  def test_build_run_flags_for_model(self):
+    model = common_definitions.Model(
+        id="1234",
+        name="tflite_m",
+        tags=[],
+        source_type=common_definitions.ModelSourceType.EXPORTED_TFLITE,
+        source_url="https://example.com/xyz.tflite",
+        entry_function="main",
+        input_types=["1xf32", "2x2xf32"])
+
+    flags = run_module_utils.build_run_flags_for_model(
+        model, common_definitions.ZEROS_MODEL_INPUT_DATA)
+
+    self.assertEqual(flags, [
```

Comment:
Does order matter? If not, use `assertCountEqual` (name is confusing IMO, but it's say the two sequences have equal counts of each item)

Reply:
In some cases the order does matter (e.g. multiple `--function_input=` should be in order). I'll keep the `assertEqual` to make sure any change on the order is verified.